application/extractors.go

```diff
@@ -34,14 +31,6 @@ var URL Extractor = func(key string, req *Request) (interface{}, error) {
 		return nil, fmt.Errorf("URL %s is not valid", value)
 	}
 
-	mimetype := mime.TypeByExtension(filepath.Ext(value))
-
-	_, ok = image.Extensions[mimetype]
-
-	if !ok {
-		return nil, fmt.Errorf("Mimetype %s is not supported", mimetype)
-	}
```

Comment:
The fact is, this test was needed to avoid process useless mimetype, do you have an alternative in mind?